Configuring one chain/single validator panics when funding a user via faucet

It seems that configuring a single chain/validator leads to a panic when attempting to create and fund a user.

1. Create a new `Interchain` add a single chain, and build (skip path creation).
```go
ic := interchaintest.NewInterchain().AddChain(chainA)

err := ic.Build(ctx, s.GetRelayerExecReporter(), interchaintest.InterchainBuildOptions{
		TestName:         s.T().Name(),
		Client:           s.DockerClient,
		NetworkID:        s.network,
		SkipPathCreation: true,
	})
s.Require().NoError(err)
```

2. When a user is attempted to be funded via `interchaintest.GetAndFundTestUsers()` it leads to a panic:

```
logger.go:130: 2023-02-17T11:28:32.490+0100 INFO  About to start container  {"chain_id": "chain-a", "test": "TestUpgradeTestSuite/TestChainUpgrade", "image": "damiannolan/sdk-simd:v0.46.9", "test_name": "TestUpgradeTestSuite/TestChainUpgrade", "command": "simd tx bank send faucet cosmos1elw6h8uzglrnlwpfhjmcxrxhc43gwz07wddszz 100000000atoma --from faucet --gas-prices 0.00atoma --gas-adjustment 1.3 --keyring-backend test --output json -y --home /var/cosmos-chain/simapp-a --node tcp://chain-a-val-0-TestUpgradeTestSuite_TestChainUpgrade:26657 --chain-id chain-a", "hostname": "TestUpgradeTestSuite_TestChainUpgrade-bnyydh", "container": "TestUpgradeTestSuite_TestChainUpgrade-bnyydh"}
panic: runtime error: index out of range [0] with length 0

goroutine 134 [running]:
github.com/strangelove-ventures/interchaintest/v7/chain/cosmos.(*CosmosChain).getFullNode(0x130?)
        /Users/damiannolan/go/pkg/mod/github.com/strangelove-ventures/interchaintest/v7@v7.0.0-20230201060741-e0e4e0f9283b/chain/cosmos/cosmos_chain.go:181 +0x10d
github.com/strangelove-ventures/interchaintest/v7/chain/cosmos.(*CosmosChain).Height(0xc0002fc640?, {0x102d44998, 0xc00178d180})
        /Users/damiannolan/go/pkg/mod/github.com/strangelove-ventures/interchaintest/v7@v7.0.0-20230201060741-e0e4e0f9283b/chain/cosmos/cosmos_chain.go:810 +0x25
github.com/strangelove-ventures/interchaintest/v7/testutil.(*height).WaitForDelta(0xc000d12f48, {0x102d44998, 0xc00178d180}, 0x2)
        /Users/damiannolan/go/pkg/mod/github.com/strangelove-ventures/interchaintest/v7@v7.0.0-20230201060741-e0e4e0f9283b/testutil/wait.go:89 +0x69
github.com/strangelove-ventures/interchaintest/v7/testutil.WaitForBlocks.func1()
        /Users/damiannolan/go/pkg/mod/github.com/strangelove-ventures/interchaintest/v7@v7.0.0-20230201060741-e0e4e0f9283b/testutil/wait.go:27 +0x57
golang.org/x/sync/errgroup.(*Group).Go.func1()
        /Users/damiannolan/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
        /Users/damiannolan/go/pkg/mod/golang.org/x/sync@v0.1.0/errgroup/errgroup.go:72 +0xa5
FAIL    github.com/cosmos/ibc-go/e2e/tests/upgrades     61.852s
FAIL
```

Seems to be coming from here -> https://github.com/strangelove-ventures/interchaintest/blob/main/chain/cosmos/cosmos_chain.go#L182

--- a/chain/cosmos/cosmos_chain.go
+++ b/chain/cosmos/cosmos_chain.go
@@ -194,6 +194,10 @@ func (c *CosmosChain) getFullNode() *ChainNode {
 	return c.Validators[0]
 }
 
+func (c *CosmosChain) GetNode() *ChainNode {
+	return c.Validators[0]
+}
+
 // Exec implements ibc.Chain.
 func (c *CosmosChain) Exec(ctx context.Context, cmd []string, env []string) (stdout, stderr []byte, err error) {
 	return c.getFullNode().Exec(ctx, cmd, env)
@@ -390,8 +394,8 @@ func (c *CosmosChain) PushNewWasmClientProposal(ctx context.Context, keyName str
 		return tx, "", err
 	}
 	message := wasmtypes.MsgStoreCode{
-		Signer: types.MustBech32ifyAddressBytes(c.cfg.Bech32Prefix, authtypes.NewModuleAddress(govtypes.ModuleName)),
-		WasmByteCode:   content,
+		Signer:       types.MustBech32ifyAddressBytes(c.cfg.Bech32Prefix, authtypes.NewModuleAddress(govtypes.ModuleName)),
+		WasmByteCode: content,
 	}
 	msg, err := c.cfg.EncodingConfig.Codec.MarshalInterfaceJSON(&message)
 	if err != nil {

